DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -704,19 +728,46 @@ function TimerMapFixes()
         break;
 
     case "14_Oceanlab_silo":
-        prevMapsDone = dxr.flagbase.GetBool('Heliosborn') &&  //Finished Vandenberg, mission 12
-            dxr.flagbase.GetBool('schematic_downloaded'); //Finished Ocean Lab, mission 14,
-        prevMapsDone = prevMapsDone || !#defined(injections) || dxr.flags.settings.goals<=0;
+        // by design, no infolink plays after killing Howard Strong if the missile hasn't been redirected
 
-        if(prevMapsDone
-           && dxr.flagbase.GetBool('missile_launched') //Redirected the missile in Silo, mission 14
-           && !dxr.flagbase.GetBool('MS_HowardStrongUnhidden'))
-        {
-            foreach AllActors(class'#var(prefix)HowardStrong', hs) {
-                hs.EnterWorld();
+        if (!dxr.flagbase.GetBool('DXR_SiloEscapeHelicopterUnhidden') && dxr.flagbase.GetBool('missile_launched')) {
+            foreach AllActors(class'HowardStrong', hs) {
+                strongAlive = True;
+                break;
+            }
+
+            if (strongAlive == False) {
+                // both goals completed just now
+                foreach AllActors(class'BlackHelicopter', chopper, 'BlackHelicopter') {
+					chopper.EnterWorld();
+                    break;
+                }
+				dxr.flagbase.SetBool('DXR_SiloEscapeHelicopterUnhidden', True,, 15);
+            } else {
+                // only computer goal completed
+                player().StartDataLinkTransmission("DL_Savage3");
             }
-            dxr.flagbase.SetBool('MS_HowardStrongUnhidden', True,, 15);
         }
+
+        if (dxr.flagbase.GetBool('DXR_SiloEscapeHelicopterUnhidden')) {
+            if (dxr.flagbase.GetBool('DL_Savage3_Played')) {
+                // both goals completed, computer infolink played
+                player().StartDataLinkTransmission("DL_Dead");
+            } else {
+                // both goals completed, computer infolink not played
+                for (ce = GetConversation('DL_Savage3').eventList; ce != None; ce = ce.nextEvent) {
+                    if (
+                        ce.nextEvent != None &&
+                        ce.nextEvent.eventType == ET_Speech &&
+                        ConEventSpeech(ce.nextEvent).conSpeech.speech == "JC, go to the silo.  Make sure no one interferes with this launch."
+                    ) {
+                        ce.nextEvent = None;
```

Comment:
Can't just change ce.nextEvent to None, you have to stitch the nextEvent to ce.nextEvent.nextEvent instead (so that it hits the ET_End event afterwards)